src/tacacs/pam/0008-Extract-tacacs-support-functions-into-library.patch

```diff
@@ -1,21 +1,21 @@
-From d820001f60e0a9f5e5df83b1edb229be5212e0b5 Mon Sep 17 00:00:00 2001
+From 834d439c8df3574d54ba88fe6cae3fbd61e4ac1f Mon Sep 17 00:00:00 2001
 From: liuh-80 <58683130+liuh-80@users.noreply.github.com>
 Date: Tue, 12 Oct 2021 10:09:10 +0800
-Subject: [PATCH 3/4] Extract tacacs support functions into library.
+Subject: [PATCH] Extract tacacs support functions into library.
```

Comment:
>Extract tacacs support functions into library. [](http://example.com/codeflow?start=17&length=46)

What is the intention of changing this file? Is it relating to src/tacacs/bash_tacplus?

Reply:
There are some code bug in this patch file:
1. When parse nss_tacplus.conf, can't get TACACS server passkey, because the file format little different with upstream file format.
2. Not release tacacs server data before load tacacs config file.

Reply:
For the code change in this file, please check this PR: https://github.com/liuh-80/pam_tacplus/pull/4/files

Reply:
1 comments about code position fixed in this patch file.
the other comments about debug code change replied, it's necessary for debug.

Reply:
According to discussion, update the patch and this code to only support debug, which is the upstream project beahvior.

Reply:
Discussed offline: we will not use pam_tacplus function to parse nss_tacplus config in future. Then we will abandon this patch.